resolwe/flow/management/commands/runexecutor_listener.py

```diff
@@ -0,0 +1,37 @@
+""".. Ignore pydocstyle D400.
+
+=================================
+Standalone Executor Contact Point
+=================================
+
+Command to run on local machine::
+
+    ./tests/manage.py runexecutor_listener
+
+"""
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from resolwe.flow.managers.listener import ExecutorListener
+
+
+class Command(BaseCommand):
+    """Run the executor listener."""
+
+    help = 'Run the standalone manager contact point for executors.'
+
+    def add_arguments(self, parser):
+        """Add command arguments."""
+        super(Command, self).add_arguments(parser)
+        parser.add_argument('--clear-queue', action='store_true',
+                            help='Consume and ignore any outstanding messages in the result queue on startup.')
+
+    def handle(self, *args, **kwargs):
+        """Run the executor listener. This method never returns."""
+        listener = ExecutorListener(redis_params=settings.FLOW_MANAGER.get('REDIS_CONNECTION', {}))
+        if kwargs['clear_queue']:
+            listener.reset()
```

Comment:
Any reason why the operation is called `clear_queue` while the method is called `reset`? I think it would be better to unify these.

Reply:
Both are `clear_queue` now.